src/components/templates/PosterPage/components/PosterPageSettingsControl/PosterPageSettingsControl.scss

```diff
@@ -0,0 +1,19 @@
+@import "scss/constants";
+
+.PosterPageSettingsControl {
+  margin-top: 150px;
+  &__popover-content {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    height: 100%;
+    width: 100%;
+
+    padding: $spacing--sm $spacing--md;
+  }
+}
+// Increase specificity to override third-party lib styles
+div.PosterPageSettingsControl__popover {
+  height: 100px;
+}
```

Comment:
Increase can be done in other ways, even `!important`, while also not prepending with `div`.
Another way is to just not use the class from 3rd party libraries.

Yet, the easiest way is to just put one more class inside `.PosterPageSettingsControl` just like every other component does.

Reply:
removed as it's unnecessary c90aa28